css/build.css

```
@@ -0,0 +1,3 @@
+fl-helper[name="image-component"] .font-weight-bold {
```

Remove all use of `component`

```suggestion
fl-helper[name="image"] .font-weight-bold {
```